### PR TITLE
Adding methods

### PR DESCRIPTION
**Pull request by joer9514**
Hey Bro! added two methods in your code!

### CODE REVIEW

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -1,76 +1,98 @@
...
+        Session = scoped_session(sess_factory)
+        self.__session = Session
+
+    def close(self):
+        """call remove() method on the private session attribute"""
+        self.__session.remove()
+
+    def get(self, cls, id_):
+        """
+        A method to retrieve one object:
```
Comment (FabianMosquera) [inline models/engine/db_storage.py:80]: Docstrings: is it enough to understand the code?

File: models/engine/db_storage.py [hunk 2]
```diff
@@ -1,76 +1,98 @@
...
+        A method to count the number of objects in storage:
+        """
+        if cls is None:
+            total = 0
+            for v in self.__models_available.values():
+                total += self.__session.query(v).count()
+            return total
+        if cls in self.__models_available.keys():
+            return self.__session.query(self.__models_available[cls]).count()
+        return -1
```
Comment (FabianMosquera) [inline models/engine/db_storage.py:98]: If return "none" affects the output of the code?

**Comment by FabianMosquera** [comment]
Hey Bro! i review your code there are 2 lines that generate questions, rest there are no drawbacks

File: models/engine/db_storage.py (hunk 1 above)
Comment (joer9514) [reply models/engine/db_storage.py:80]: Yes, the documentation says what its functionality is.

File: models/engine/db_storage.py (hunk 2 above)
Comment (joer9514) [reply models/engine/db_storage.py:98]: Obviously it would change the output and not return any value.

**Comment by joer9514** [comment]
I hope to have solved your doubts, a greeting!

**Pull request merged**